build.sh

```diff
@@ -9,3 +9,25 @@ CONTAINERS_STORAGE_THIN_TAGS="containers_image_openpgp exclude_graphdriver_btrfs
 cd bib
 set -x
 go build -tags "${CONTAINERS_STORAGE_THIN_TAGS}" -o ../bin/bootc-image-builder ./cmd/bootc-image-builder
+
+# expand the list as we support more architectures
+for arch in amd64 arm64; do
+    if [ "$arch" = "$(go env GOARCH)" ]; then
+	continue
+    fi
+
+    # what is slightly sad is that this generates a 1MB file. We could do
```

Comment:
Yeah, I originally wrote this in C but then found that there is only a cross gcc in fedora for building a kernel and nothing userspace and I couldn't even get: `void _start() { syscall(SYS_exit() }`to build so that did not work out. Rust is a super interesting idea it would mean a manual rustup in the containerfile, right? the rust rpms do not have cross build support (?). Maybe a (fun) followup :)